### DISCUSSION TITLE
Curation Week October 10-14

### DISCUSSION THREAD

**Discussion by elissonnog** [wikipathways/GPML-Repo#95] [Curation reports]
Hi team,
Here is my report:

-Featured
WP4351, WP5284, WP5285
-Approved Version
WP5277, WP5279, WP5276,
- Under Construction
WP5242,
-Needs Work
WP5218, WP5288, WP5078, WP5280, WP5286, WP5287, WP5289

There's some good pathways but seems to be abandoned. 
- Deletion requested
WP5259, WP5263, WP5236, WP5077

PS:
There is an atypical error message on this pathway (WP5078) tag: "Expression error: Unrecognised punctuation character "{""

The search works fine.

Best, 
Elisson

**Comment by AlexanderPico** [comment]
Thanks, Elisson!  I looked into the tag error, but couldn't identify the source of the problem. This particular Curation Tag works fine on other pathway (with current and older revisions tagged) and I even tried removing it and resetting it, but the error remains.

Good news: We'll be switching to the new site soon and all that old Curation Tag code will be tossed in the bin :)

**Comment by mkutmon** [comment]
@elissonnog is there a specific reason why you removed the analysis tag of WP5078? I added it to the analysis collection yesterday and I am not sure what you think is missing at this point. Can you clarify? 
I will look into the issue with the tag but as Alex mentioned - not super important anymore :)

**Reply by elissonnog** [reply]
Hi Martina.
I removed because of the miss annotated interactions.
I am sorry if I did wrong.

**Reply by mkutmon** [reply]
No worries! I just wanted to make sure I am not missing something. I don't think there are any unconnected lines but I will double-check and add the tag again. 
Thanks for checking it so carefully! But especially for enrichment analysis and visualization, this pathway is already quite valuable so it's okay to add it to the collection.
